packages/data-transformation/fhir-to-csv/src/parseFhir/parseFhir.py

```
@@ -0,0 +1,341 @@
+                    thisRow[i] = combineValues(anchorArray,
+                                               paths[i][7:]
+                                               , filename)
+                else:
+                    thisRow[i] = combineValues(jsndict, paths[i], filename)
+
+            writerow_flex(data, csvwriter, thisRow, outputFormat)
+            return 1
+
+
+def parse(configPath,inputPath=None,outputPath=None,missingPath=None,outputFormat=None,inputFormat=None,writeMode=None):
+    f = open(configPath, "r")
```

Ruff (0.12.2)

Local variable `f` is assigned to but never used

Remove assignment to unused variable `f`

(F841)